Classes/DDLoggerNames.h

```diff
@@ -0,0 +1,30 @@
+// Software License Agreement (BSD License)
+//
+// Copyright (c) 2010-2018, Deusty, LLC
+// All rights reserved.
+//
+// Redistribution and use of this software in source and binary forms,
+// with or without modification, are permitted provided that the following conditions are met:
+//
+// * Redistributions of source code must retain the above copyright notice,
+//   this list of conditions and the following disclaimer.
+//
+// * Neither the name of Deusty nor the names of its contributors may be used
+//   to endorse or promote products derived from this software without specific
+//   prior written permission of Deusty, LLC.
+
+#import <Foundation/Foundation.h>
+
+typedef NSString *const DDLoggerName NS_STRING_ENUM;
+FOUNDATION_EXPORT DDLoggerName DDLoggerNameASL; // DDASLLogger
+FOUNDATION_EXPORT DDLoggerName DDLoggerNameTTY; // DDTTYLogger
+FOUNDATION_EXPORT DDLoggerName DDLoggerNameOSDefault; // DDOSLogger
+FOUNDATION_EXPORT DDLoggerName DDLoggerNameFile; // DDOSLogger
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
This should come before the typedef. Otherwise we don't have nullability for the logger names.